lib/agents/maige.ts

```diff
@@ -37,11 +38,20 @@ export async function maige({
 	issueId?: string
 	pullUrl?: string
 	allLabels: any[]
+	comment: any
 	beta?: boolean
 }) {
 	const tools = [
 		labelTool({octokit, allLabels, issueId}),
-		updateInstructionsTool({octokit, prisma, customerId, issueId, repoFullName}),
+		updateInstructionsTool({
+			octokit,
+			prisma,
+			customerId,
+			issueId,
+			repoFullName,
+			instructionCreator: comment.user.login,
+			instructionCommentLink: comment.html_url
```

Comment:
Nice! Could we do without `instructionCommentLink` since `html_url` can be inferred from `user.login`?